will-schulz.txt

```diff
@@ -0,0 +1 @@
+My name is Will.
```

Comment:
Nice... you got everything good.  Now, when you finish with the MVP goals I will be able to confirm by closing the pull request which will merge your branch back into the main.  Then if you want to do more work make a new branch since that will then leave your "production ready" code untouched as you experiment and "develop" more.